source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/activity/edit/InvoiceTotalsForm.tsx

```diff
@@ -39,7 +44,16 @@ export const InvoiceTotalsForm: React.FunctionComponent<
   }, [setFieldValue, values.invoices, values.totalAmount]);
 
   return (
-    <Section header="Invoices Total">
+    <Section
+      header={
+        <SimpleSectionHeader title="Invoices Total">
+          <StyledSectionAddButton onClick={onAdd}>
+            <FaPlus size="2rem" />
+            &nbsp;{'Add an Invoice'}
```

Comment:
nit: not a fan of the '&nbsp;'

Reply:
Me neither but that is prettier doing auto-formatting. I removed it and the auto-formatted put it back